apps/web/pages/organizations/[organizationId]/invoices/[invoiceId]/components/invoiceDetails.tsx

```diff
@@ -88,11 +92,16 @@ export const InvoiceDetails = ({ invoice, invoiceItems }: InvoiceDetailsProps) =
           </tr>
         </tbody>
       </table>
-      <div className="flex flex-col">
-        <p className="text-sm font-bold">
-          Payment method: <span className="font-normal">Bank Transfer/ PayPal</span>
-        </p>
-        <p className="text-sm">Thank you for your business!</p>
+      <div className="flex">
+        <div className="flex flex-col">
+          <p className="text-sm font-bold">
+            Payment method: <span className="font-normal">Bank Transfer/ PayPal</span>
+          </p>
+          <p className="text-sm">Thank you for your business!</p>
+        </div>
+        <div className="w-full text-right font-bold">
+          Invoice status: <span className="font-normal">{invoiceData.invoiceStatus}</span>
+        </div>
```

Comment:
How about moving this inside a invoice status badge component and put it next to `Invoice` on line 61, for example:
![image](https://github.com/user-attachments/assets/056258a9-9e32-4764-ac92-c434ebce0998)

The way its implemented in the screenshot :

```
          <div>
            <div className="flex items-center gap-2">
              <div className="text-2xl font-bold">Invoice</div>
              <span className="badge badge-neutral badge-lg">{invoiceData.invoiceStatus}</span>
            </div>
            <p className="text-sm text-gray-600">Invoice No: #{invoiceData.id}</p>
            <p className="text-right text-sm text-gray-600">Invoice Date: {invoiceData.invoiceDate}</p>
          </div>
 ```

Reply:
Good idea! Implemented it.